baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -406,14 +403,47 @@ def hashes_comply(s1, s2):
                 {"url": None, "block_hash": None},
             )
 
-            self.config["snapshot_url"] = snapshot_metadata["url"]
-            self.config["snapshot_sha256"] = snapshot_metadata.get("sha256", None)
-            self.config["snapshot_block_hash"] = snapshot_metadata["block_hash"]
+            self.config["snapshot_metadata"] = snapshot_metadata
+
         except (urllib.error.URLError, ValueError):
             print(f"Couldn't collect snapshot metadata from {json_url}")
         except Exception as e:
             print(f"Unexpected error handling snapshot metadata:\n{e}\n")
 
+    def output_snapshot_metadata(self):
+        from datetime import datetime
+        from locale import setlocale, getlocale, LC_TIME
+
+        # it is portable `C` locale by default
+        setlocale(LC_TIME, getlocale())
```

Comment:
Maybe I'm missing something, but why is this necessary?

Shouldn't the locale already be set on the machine? :thinking:

Reply:
It is set on machine, but python still trying to use portable locale by default
After that change, date became french on my `fr_FR.UTF-8` localized machine

Reply:
Ok, I see that apparently this sets the global locale of the python app, which should be fine here.